api/controllers/commentController.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const app = express();
+const Comment = require('../Schema/commentSchema')
+const User = require('../Schema/userSchema')
+
+app.route('/:id/comments')
```

Comment:
You did not put routes directly in the controller. Instead, you only write middleware in the controller and separate routes in another file.

Reply:
Fixed